contentcuration/contentcuration/view/internal_views.py

```diff
@@ -401,20 +401,23 @@ def get_channel_status_bulk(request):
     """ Create the channel node """
     data = json.loads(request.body)
     try:
-        statuses = {}
-        for channel_id in data['channel_ids']:
-            statuses[channel_id] = get_status(channel_id)
+        statuses = {cid: get_status(cid) for cid in data['channel_ids']}
 
-        return HttpResponse(json.dumps({"success": True, 'statuses': statuses}))
+        return HttpResponse(json.dumps({
+            "success": True,
+            'statuses': statuses,
+        }))
 
     except KeyError:
         raise ObjectDoesNotExist("Missing attribute from data: {}".format(data))
     except Exception as e:
         return HttpResponseServerError(content=str(e), reason=str(e))
 
 def get_status(channel_id):
-    obj = Channel.objects.get(pk=channel_id)
-    if obj.deleted:
+    obj = Channel.objects.filter(pk=channel_id).first()
+    if not obj:
+        return "active"
```

Comment:
So, if I am reading this right, if someone queries any arbitrary channel_id that doesn't exist yet, this will just return saying it is `active`? But, because we are only expecting this to be queried from the frontend of Studio, we would only query a non-existent channel if it was still in the process of being created?

Reply:
Yes, this endpoint is only used by the sushibar, and "active" is the default state. I can update the sushibar to handle a "non-existent" state though. @ivanistheone thoughts?